src/sorcha_cmdline/cite.py

```diff
@@ -0,0 +1,57 @@
+#
+# The `sorcha cite` subcommand implementation
+#
+import argparse
+from sorcha_cmdline.sorchaargumentparser import SorchaArgumentParser
+import os
+
+
+def main():
+    parser = SorchaArgumentParser(
+        prog="sorcha cite",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description="Providing the bibtex, AAS Journals software latex command, and acknowledgement statements for Sorcha and the associated packages that power it in a file called sorcha_citation.txt.",
+    )
+
+    optional = parser.add_argument_group("Optional arguments")
+    optional.add_argument(
+        "-o",
+        "--outfile",
+        help="File path to store citation file sorcha_citation.txt.",
+        type=str,
+        dest="o",
+        required=False,
+        default=os.getcwd(),
+    )
+
+    optional.add_argument(
+        "-p",
+        "--print",
+        help="Prints citation to terminal.",
+        dest="p",
+        action="store_true",
+        required=False,
+    )
+
+    args = parser.parse_args()
+
+    return execute(args)
+
+
+def execute(args):
+    import sys
+    from sorcha.utilities.citation_text import cite_sorcha
+
+    if args.p:  # prints citation to terminal
+        cite_sorcha()
+
```

Comment:
it looks if I have with -p it will execute both printing to a file and to the terminal